src/main/java/frc/robot/autonomous/AutonomousConstants.java

```diff
@@ -11,4 +14,11 @@ public class AutonomousConstants {
 
 	public static final double CLOSE_TO_TARGET_POSITION_DEADBAND_METERS = 0.5;
 
+	public static final RobotConfig SYNCOPA_ROBOT_CONFIG = new RobotConfig(
+		74,
+		8.6,
+		new ModuleConfig(0.048, 5.24, 0.96, DCMotor.getFalcon500Foc(1), 60, 1),
+		0.577
+	);
+
```

Comment:
All these numbers need to be stored as separate constants, and then applied to the constant RobotConfig

- [ ] move all numbers to separate constants